app.rb

```diff
@@ -16,5 +16,6 @@ class RockPaperScissors < Sinatra::Base
     erb :gameplay
   end
 
+
   run! if app_file == $0
 end
```

Comment:
I found the naming conventions easy to understand, I was able to identify clearly the route the user would take as they navigated through the interface.